Splitio-net-core.Integration-tests/BaseIntegrationTests.cs

```diff
@@ -1041,7 +1041,7 @@ public void Destroy()
             var splitFactory = new SplitFactory(apikey, configurations);
             var client = splitFactory.Client();
 
-            client.BlockUntilReady(10000);
+            client.BlockUntilReady(10000);            
```

Comment:
could you please get rid of those spaces?

Reply:
Ready! thanks for your feedback @mmelograno